Scalar.Common/Platforms/POSIX/POSIXPlatform.cs

```diff
@@ -142,6 +142,23 @@ public override NamedPipeServerStream CreatePipeByName(string pipeName)
             return pipe;
         }
 
+        public override string GetSecureDataRootForScalar()
+        {
+            // SecureDataRoot is Windows only. On POSIX, it is the same as CommoAppDataRoot
+            return this.GetCommonAppDataRootForScalar();
+        }
+
+        public override string GetSecureDataRootForScalarComponent(string componentName)
+        {
+            // SecureDataRoot is Windows only. On POSIX, it is the same as CommoAppDataRoot
```

Comment:
Same (existing) typo here.

```suggestion
            // SecureDataRoot is Windows only. On POSIX, it is the same as CommomAppDataRoot
```

Reply:
Yup, this is an exact copy from the originals, which I left mostly in case anyone was searching with grep or doing other comparisons, but I'm happy to correct it also.